wallet/walletrpc.c

```diff
@@ -807,9 +807,9 @@ static const struct json_command listaddrs_command = {
 };
 AUTODATA(json_command, &listaddrs_command);
 
-static struct command_result *json_outputs(struct command *cmd,
-					   struct json_stream *response,
-					   struct utxo **utxos)
+static void json_add_utxos(struct command *cmd,
```

Comment:
Other `json_add_*` functions get `(struct json_stream *response, const char *field, const _whatever_ *value)` as arguments, would it not be better to use similar convention?

Reply:
Agreed, with the one exception that we need access to the log here, which is why `cmd` is being passed in as well. Would be better to pass in the more specific `log` instead of the `cmd` instance, which gives access to loads of stuff that this function shouldn't have access to.

Reply:
Actually, this arg is replaced in the next patch with "wallet", where we do appropriate logging.

Reply:
But I've moved it to the end, for consistency as @ZmnSCPxj points out.